nominees/ekshop.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "ekShop",
+  "aliases": [
+    ""
+  ],
+  "description": "ekShop is a fully open-source eCommerce platform that is customizable and configurable to your needs. It is completely free, adaptable and open to be supported by a worldwide community of volunteers and contributors. It’s free and open source nature allows users to maintain complete control of the data content and modify it as they wish and according to their needs. Our intended goal is to allow any potential entrepreneur to quickly get up and running with an online platform to sell goods online.",
+  "website": "https://ekshop.gov.bd/",
+  "license": [
+    {
+      "spdx": "Apache-2.0",
+      "licenseURL": "https://github.com/a2i-dpg/ekShop/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 1,
+      "evidenceText": "ekShop is a scalable open-source eCommerce solution that provides the power of harnessing online marketplace and businesses into the hands of everyday folks. ekShop allows people to bring forth sustainable livelihoods by engaging entrepreneurs, merchants, and employees that help the business operate. As a result, ekShop, when implemented correctly, can uplift communities and populations out of poverty."
```

Comment:
Does an e-commerce platform really contribute to reducing poverty? This feels like a stretch to me. Perhaps it could be debated whether it aligns to [Target 1.4 of the SDG](https://sdgs.un.org/goals/goal1), but given the indicators defined for that target (below), I am struggling to see the connection. Just making an e-commerce platform into something open source does not mean it will automatically go into the hands of everyday folks as they described here. That takes effort, and that effort should have some evidence, which I am not seeing.

![Screenshot of Target 1.4 and its indicators for SDG 1 (No Poverty)](https://user-images.githubusercontent.com/4721034/153402236-15306370-cdf7-42e6-b40b-d9ffcd67c943.png "Screenshot of Target 1.4 and its indicators for SDG 1 (No Poverty)")

Reply:
Resolved in commit 6cb896137447acf8bbb26f5cdc9691d7001fd7e3.